player.js

```diff
@@ -0,0 +1,63 @@
+var Scrabble = require("./scrabble.js");
+var scrabble = new Scrabble();
+
+var Player = function(name) {
+  this.name =  name;
+  this.plays = [];
+};
+
+Player.prototype.play = function(word) {
+  if (this.hasWon()) {
+    return false
+  }
+  this.plays.push(word);
+}
```

Comment:
You probably need a semicolon at the end of each of your prototype statements.  The rules aren't very clear, imho, so 100% trust me on that.  The linter (mentioned below) that we use at my internship will tell you need on here. Gah...so confusing!